std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -111,34 +111,38 @@ struct KRRegion(ParentAllocator = NullAllocator)
 
         this(this) @disable;
 
+        nothrow @nogc @safe
         void[] payload() inout
         {
-            return (cast(ubyte*) &this)[0 .. size];
+            return (() @trusted => (cast(ubyte*) &this)[0 .. size])();
```

Comment:
Better not create lambdas to not hinder performance by calling an unnecessary function. You can make this function itself `@trusted`.